pkg/ddc/alluxio/dataset.go

```diff
@@ -118,6 +118,13 @@ func (e *AlluxioEngine) UpdateDatasetStatus(phase datav1alpha1.DatasetPhase) (er
 		datasetToUpdate.Status.CacheStates = runtime.Status.CacheStates
 		// datasetToUpdate.Status.CacheStates =
 
+		if datasetToUpdate.Status.HCFSStatus == nil {
+			datasetToUpdate.Status.HCFSStatus, err = e.GetHCFSStatus()
```

Comment:
I think we should clear HCFSStatus when deleting related runtime

Reply:
Thanks for your suggestion.